sunpy/visualization/animator/line.py

```diff
@@ -77,7 +78,7 @@ class LineAnimator(ArrayAnimator):
           In this scenario the same x-axis values are used in every frame of the animation.
        b) Second, the x-axis array can have the same shape as the data array.
           In this scenario the x-axis is refreshed for each frame. For example, if
-          ``data.shape == axis_ranges[plot_axis_index] == (4, 3)``,
+          ``data.shape == axis_ranges[plot_axis_index] == (4, 3)``,     #shouldn't it be axis_ranges[plot_axis_index].shape ?
```

Comment:
Can you check this?

Reply:
Yeah, I was going to say, I forgot the comment there.

Reply:
If you don't mind, I'll check this about 14 hours later.? I'm a bit occupied now.

Reply:
Sure, there is no rush.

Reply:
Yes, it should be as I expect it.